tethys_apps/base/permissions.py

```diff
@@ -102,6 +100,15 @@ def __repr__(self):
         return self._repr()
 
 
+def _has_permission(app, user, perm):
+    namespaced_perm = f"tethys_apps.{app.package}:{perm}"
```

Comment:
What's the difference between `app.package` and `app.package_namespace`? Should we be using one over the other?

Reply:
The `package_namespace` property is `tethysapp` for `TethysAppBase` and `tethysext` for `TethysExtensionBase`.  The `app.package` property is specific to a particular app and is what should be used here.